docs/SwiftFormat.md

```diff
@@ -0,0 +1,49 @@
+
+# Swift-format
+
+## Introduction
+
+Note: This tool is still a work in progress.
+
+Swift-format is a tool for automatically format your Swift files according to a
```

Comment:
Should this perhaps be spelt as the tool name (`swift-format`) rather than using the sentence casing?